src/main/java/Duke.java

```diff
@@ -1,10 +1,76 @@
+import java.util.Objects;
+import java.util.Scanner;
+
 public class Duke {
+    public static String concatenateChar(String line, String start, String end) {
+        int indexOfStart = line.indexOf(start) + 1;
+        int indexOfEnd = 0;
```

Comment:
Probably modifying variable names to "endIndex" and "startIndex" improves readability